src/App.css

```diff
@@ -1,38 +1,21 @@
-.App {
-  text-align: center;
-}
-
-.App-logo {
-  height: 40vmin;
-  pointer-events: none;
-}
 
```

Comment:
Could you please move your styles into a separate `InputSearchBox.css` file?

Reply:
Sure, no problem. I’ll do it as soon as I can, because I’m on a seaside right now, so I don’t have an access to my PC. Sorry for some inconveniences. Today I don’t able to join on discord server. If you have any questions then leave a comment here or write a message in telegram. Tomorrow I’ll return home.